app.js

```diff
@@ -13,69 +13,89 @@ const teamOneLabel = document.getElementById('team-one-name');
 const teamTwoLabel = document.getElementById('team-two-name');
 
 // create an array to hold on to the state of past games
-
-let name1 = '',
-let name2 =  '',
-let score1 = 0,
-let score2 = 0,
+const pastGamesArray = [];
+let name1 = '';
+let name2 = '';
+let score1 = 0;
+let score2 = 0;
 
 nameForm.addEventListener('submit', (e) => {
     // don't forget to prevent the default form behavior!
-
+    e.preventDefault();
     // get the name data from the form
-
+    const data = new FormData(nameForm);
     // set the state to this data from the form
-
+    name1 = data.get('name-1');
+    name2 = data.get('name-2');
```

Comment:
Look at the names for the inputs in the form in index.html.  They should match when you are passing into `data.get` on 28 and 29.